### PR TITLE
Get the correct property from pg query start channel

### PR DESCRIPTION
**Pull request by CarlesDD**
### What does this PR do?
Get the correct value in `apm:pg:query:start` channel subscription to analyze the query.

### Motivation
The message published by `pg` instrumentation has changed, so `statement` property is no longer present in the message in favor of `pgQuery` object. The changes in this PR extracts the query from the received object to be analyzed by `SQLi analyzer`.

### CODE REVIEW

File: packages/dd-trace/src/appsec/iast/analyzers/sql-injection-analyzer.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ class SqlInjectionAnalyzer extends InjectionAnalyzer {
     super('SQL_INJECTION')
     this.addSub('apm:mysql:query:start', ({ sql }) => this.analyze(sql))
     this.addSub('apm:mysql2:query:start', ({ sql }) => this.analyze(sql))
-    this.addSub('apm:pg:query:start', ({ statement }) => this.analyze(statement))
+    this.addSub('apm:pg:query:start', ({ query }) => this.analyze(query.text))
```
Comment (iunanua) [inline packages/dd-trace/src/appsec/iast/analyzers/sql-injection-analyzer.js:9]: I one day it changes again and `query` is `undefined`, `query.text` will throw an Error.

Comment (CarlesDD) [reply packages/dd-trace/src/appsec/iast/analyzers/sql-injection-analyzer.js:9]: It's actually checked in the [instrumentation](https://github.com/DataDog/dd-trace-js/blob/master/packages/datadog-instrumentations/src/pg.js#L36)

IMO, publishing a message in a query:start channel without the query object has no sense, but we can re-check it here for redundancy.

**Review by hoolioh — APPROVED** [review]

**Review by uurien — APPROVED** [review]

File: packages/dd-trace/src/appsec/iast/analyzers/sql-injection-analyzer.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ class SqlInjectionAnalyzer extends InjectionAnalyzer {
     super('SQL_INJECTION')
     this.addSub('apm:mysql:query:start', ({ sql }) => this.analyze(sql))
     this.addSub('apm:mysql2:query:start', ({ sql }) => this.analyze(sql))
-    this.addSub('apm:pg:query:start', ({ statement }) => this.analyze(statement))
+    this.addSub('apm:pg:query:start', ({ query }) => query && this.analyze(query.text))
```
Comment (rochdev) [inline packages/dd-trace/src/appsec/iast/analyzers/sql-injection-analyzer.js:9]: Is this not tested? 🤔

Comment (uurien) [reply packages/dd-trace/src/appsec/iast/analyzers/sql-injection-analyzer.js:9]: I didn't know that we have a way to do integration test using the real plugins. I've created new PR, adding a way to apply plugin test to iast related test:

https://github.com/DataDog/dd-trace-js/pull/2565

**Pull request merged**